validator/testcases/markup/markuptester.py

```diff
@@ -119,18 +116,15 @@ def _feed_parser(self, line):
         line = unicodehelper.decode(line)
 
         try:
-            self.feed(line + "\n")
-        except UnicodeDecodeError, exc_instance:
-            exc_class, val, traceback = sys.exc_info()
             try:
+                self.feed(line + "\n")
```

Comment:
so you want to catch an error that would happen in this `except` body? Is that what you're saying? If you're using `ignore`, I don't think you can have exceptions. I don't see any other reason to have those nested try blocks. Am I missing something?